store/src/java/com/zimbra/cs/servlet/util/AuthUtil.java

```diff
@@ -336,4 +348,70 @@ public static String getRedirectURL(HttpServletRequest req, Server server,
         }
         return redirectUrl;
     }
+
+    public static AuthToken getAuthToken(Element request, Provisioning prov, ZimbraSoapContext zsc)
+        throws ServiceException {
+        //Fetch Auth token from cookies
```

Comment:
keep the commenting code consistent starting with small case and space after //

Reply:
Resolved and did same for other commented lines